style2.css

```diff
@@ -0,0 +1,391 @@
+@media screen and (max-width: 1440px) {
+  .main__pic {
+    margin: -100px 0 0 0 ;
+	}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)